README.md

```diff
@@ -1,4 +1,11 @@
-MurMurHash
-==========
+# MurMurHash
 
-An implementation of the MurMurHash function in C++
+An implementation of the MurMurHash3 function in C++
+
+ MurmurHash3 was originally written by Austin Appleby, and is placed in the public domain.
```

Comment:
"and is placed in the public domain" -> "and has been released into the public domain"